class.jetpack.php

```diff
@@ -4677,35 +4676,13 @@ public static function build_authorize_url( $redirect = false, $iframe = false )
 	 * - Earliest date of post of any post type.
 	 *
 	 * @since 7.2.0
+	 * @deprecated since 7.8.0
 	 *
 	 * @return string Assumed site creation date and time.
 	 */
 	public static function get_assumed_site_creation_date() {
-		$earliest_registered_users = get_users( array(
-			'role'    => 'administrator',
-			'orderby' => 'user_registered',
-			'order'   => 'ASC',
-			'fields'  => array( 'user_registered' ),
-			'number'  => 1,
-		) );
-		$earliest_registration_date = $earliest_registered_users[0]->user_registered;
-
-		$earliest_posts = get_posts( array(
-			'posts_per_page' => 1,
-			'post_type'      => 'any',
-			'post_status'    => 'any',
-			'orderby'        => 'date',
-			'order'          => 'ASC',
-		) );
-
-		// If there are no posts at all, we'll count only on user registration date.
-		if ( $earliest_posts ) {
-			$earliest_post_date = $earliest_posts[0]->post_date;
-		} else {
-			$earliest_post_date = PHP_INT_MAX;
-		}
-
-		return min( $earliest_registration_date, $earliest_post_date );
+		_deprecated_function( __METHOD__, 'jetpack-7.8', 'Automattic\Jetpack\Connection\Manager' );
```

Comment:
Oops, we should escape those because they're in single quotes:

```suggestion
		_deprecated_function( __METHOD__, 'jetpack-7.8', 'Automattic\\Jetpack\\Connection\\Manager' );
```